src/mlpack/methods/ann/init_rules/unit_init.hpp

```diff
@@ -0,0 +1,66 @@
+/**
+ * @file unit_init.hpp
+ * @author Sumedh Ghaisas
+ *
+ * Intialization rule for the neural networks. This simple initialization is
+ * performed by assigning a unit matrix (matrix of all ones) to the weight
+ * matrix.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_INIT_RULES_UNIT_INIT_HPP
+#define MLPACK_METHODS_ANN_INIT_RULES_UNIT_INIT_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * This class is used to initialize randomly the weight matrix.
+ */
+class UnitInitialization
```

Comment:
This class is only for testing right? So I think we can use the ``RandomInitialization`` class and set ``lowerBound`` and ``upperBound`` to 1. What do you think?